src/main/java/Tasks/Event.java

```diff
@@ -0,0 +1,37 @@
+package Tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class represents a task that has a duration of time which it has to be done.
+ */
+public class Event extends Task {
+
+    protected LocalDate from;
+    protected LocalDate to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = LocalDate.parse(from);
+        this.to = LocalDate.parse(to);
+    }
+
+    /**
+     * This method returns the dates and/or times that the event is due to take place
+     * in the format specified.
+     * @return the dates and/or time of the event.
+     */
+    public String getFromTo() {
+        return from.format(DateTimeFormatter.ofPattern("MMM d yyyy"))
+                + " to "
```

Comment:
I like the spacing of " to " to a new line here, makes it easier to understand which parts are hard code printed and which ones are generated/varied